src/Language/Token.php

```diff
@@ -81,18 +81,10 @@ class Token
      */
     public $prev;
 
-    /** @var Token */
+    /** @var ?Token */
     public $next;
 
-    /**
-     * @param string     $kind
-     * @param int        $start
-     * @param int        $end
-     * @param int        $line
-     * @param int        $column
-     * @param mixed|null $value
-     */
-    public function __construct($kind, $start, $end, $line, $column, ?Token $previous = null, $value = null)
+    public function __construct(string $kind, int $start, int $end, int $line, int $column, ?Token $previous = null, $value = null)
```

Comment:
missing docs for `$value`. should be `mixed` I guess

Reply:
Sorry, what is it exactly you want me to do?

Reply:
It's because there's check whether you are missing type declarations or not (currently disabled). And there would be no way to tell whether it's `mixed` or just forgoten. So `mixed` should be declared explicitely.

Reply:
also, null is included in `mixed` afaik, so `mixed` should me enough

Reply:
removed null